elem/elem.gen.go

```diff
@@ -745,48 +725,41 @@ func Sample(markup ...vecty.MarkupOrComponentOrHTML) *vecty.HTML {
 	return vecty.Tag("samp", markup...)
 }
 
-// The HTML Script Element (<script>) is used to embed or reference an
-// executable script within an HTML or XHTML document.
+// Script is used to embed or reference an executable script.
 //
 // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script
 func Script(markup ...vecty.MarkupOrComponentOrHTML) *vecty.HTML {
 	return vecty.Tag("script", markup...)
 }
 
 // Section represents a generic section of a document, i.e., a thematic
-// grouping of content, typically with a heading. Each <section> should be
-// identified, typically by including a heading (<h1>-<h6> element) as a child
-// of the <section> element.
+// grouping of content, typically with a heading.
 //
 // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section
 func Section(markup ...vecty.MarkupOrComponentOrHTML) *vecty.HTML {
 	return vecty.Tag("section", markup...)
 }
 
-// The HTML select (<select>) element represents a control that presents a menu
-// of options. The options within the menu are represented by <option>
-// elements, which can be grouped by <optgroup> elements. Options can be
-// pre-selected for the user.
+// Select represents a control that provides a menu of options.
 //
 // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select
 func Select(markup ...vecty.MarkupOrComponentOrHTML) *vecty.HTML {
 	return vecty.Tag("select", markup...)
 }
 
-// Shadow is used as a shadow DOM insertion point. You might use it if you have
-// created multiple shadow roots under a shadow host. It is not useful in
-// ordinary HTML. It is used with Web Components.
+// Slot is a placeholder inside a web component that you can fill with your own
+// markup, with the effect of composing different DOM trees together.
 //
-// https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Shadow
-func Shadow(markup ...vecty.MarkupOrComponentOrHTML) *vecty.HTML {
-	return vecty.Tag("shadow", markup...)
+// https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot
+func Slot(markup ...vecty.MarkupOrComponentOrHTML) *vecty.HTML {
+	return vecty.Tag("slot", markup...)
```

Comment:
OK, I see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Shadow is obsolete and https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot is experimental so this is correct.